error_test.go

```diff
@@ -245,6 +245,19 @@ func TestParseError(t *testing.T) {
             |         15:04:05.856018510
 			`,
 		},
+
+		{
+			&struct{ String string }{},
+			`string = "test`,
+			`
+            | toml: error: unexpected EOF; expected '"'
+			|
+			| At line 0, column 14:
+			|
+			|       0 | string = "test
```

Comment:
This is line 1, not 0.

Reply:
Sorry, I didn't notice that. I've updated the test and believe I found where the line number was being incorrectly set to 0.